test/parallel/test-fs-promises.js

```diff
@@ -102,9 +102,8 @@ function verifyStatObject(stat) {
     await chmod(dest, 0o666);
     await handle.chmod(0o666);
 
-    // Mode larger than 0o777 should be masked off.
-    await chmod(dest, (0o777 + 1));
-    await handle.chmod(0o777 + 1);
```

Comment:
I think it would be nice to check `0o777 + 1` as well (I guess it throws?).

Reply:
@BridgeAR It fails on FreeBSD, that turns the bits into `0o1000` which is `S_ISVTX`